### PR TITLE
Add Thai Gen AI (GPTeacher) dataset

### PR DESCRIPTION
**Pull request by manandey**
All tests are passing. c.c. @shayne-longpre

### CODE REVIEW

**Review by wwbrannon — APPROVED** [review]
👍

File: data_summaries/Thai Gen AI (GPTeacher).json [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+            "Multiple Choice Question Answering",
+            "Algebraic Expression Evaluation",
+            "News Headline Retrieval",
+            "Web Scraping using Python Requests",
+            "Information Retrieval",
+            "Webpage Content Retrieval",
+            "Roleplay"
+        ],
+        "Text Sources": [],
+        "Model Generated": [
```
Comment (shayne-longpre) [inline data_summaries/Thai Gen AI (GPTeacher).json:47]: Google cloud model too

File: data_summaries/Thai Gen AI (GPTeacher).json [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+            "OpenAI GPT-4"
+        ],
+        "Format": [
+            "Zero-shot"
+        ],
+        "Human Annotation": "No",
+        "Derived from Datasets": [],
```
Comment (shayne-longpre) [inline data_summaries/Thai Gen AI (GPTeacher).json:54]: gpteacher

File: data_summaries/Thai Gen AI (GPTeacher).json [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+        "Creators": [],
+        "Licenses": [
+            {
+                "License": "CC BY-SA 3.0",
```
Comment (shayne-longpre) [inline data_summaries/Thai Gen AI (GPTeacher).json:58]: does original gpteacher have licenses (just look for it in our data_summaries/)

**Review by shayne-longpre — COMMENTED** [review]
minor additions

**Pull request merged**